### PR TITLE
Alias where as filter

### PR DESCRIPTION
**Pull request by jerabaul29**
Alias where as filter to fit to the "usual" functional programming terminology. Add as a keyword for the module and an alias.

### CODE REVIEW

File: pipe.py [hunk 1]
```diff
@@ -71,6 +71,7 @@
     "transpose",
     "dedup",
     "uniq",
+    "filter"
```
Comment (JulienPalard) [inline pipe.py:74]:
```suggestion
    "filter",
```

just so the next diff won't look like:

```diff
-    "filter"
+    "filter",
+    "the new thing"
```

but simply:

```diff
+    "the new thing"
```

**Comment by jerabaul29** [comment]
Sorry I missed the comma, fixed now, should be ready to merge then :) .

**Comment by JulienPalard** [comment]
Should it be added to the readme to?

**Comment by jerabaul29** [comment]
I was waiting for the ordered list of pipes to be merged, will add it tonight :) .

**Comment by jerabaul29** [comment]
Added now, is this ok? :) . I also added the alias entry for select that I had forgotten.

**Comment by JulienPalard** [comment]
:+1: thanks!

**Pull request merged**